### ISSUE TITLE
Every time when I move an icon on the diagram the GUI is waiting for the server

### ISSUE THREAD

**Issue by 1001QAdotNET** [GNS3/gns3-gui#2036]
I guess this is updating between GUI and Server
It is slow and annoying, especially because it happens with every single device that you rearrange on the screen
Is is possible to speed that up or use batch or triggered sync ?
In the end it does not matter for the server that much to know the exact position on the canvas or whatever other cosmetic changes happen

It might be important in a multiuser (and concurent access) environment
If there is one singe user (GUI client) connected to the project make this sync in batches or even manual

**Comment by noplay** [comment]
It's not normal that is slow. Do you have an antivirus running that could intercept things?

**Comment by 1001QAdotNET** [comment]
yes I do have an antivirus but I don't think that it is a problem
as far as I can understand this is light communications and if the antivirus kicks is it should be very quick

**Comment by 1001QAdotNET** [comment]
OK the mistery was solved. 
I created the MD5sum file coresponding to the Arista vEOS image that I am playing with, while I was in root mode (su) and for some reasons the md5 file was not writable for the server. So probably the server is trying and trying and trying and in the end you see a mesage in the server log but the GUI does not see it. I still believe that the server log messages should be available somewhere in the GUI , just create two tabs in the console area, one for the GUI console and one for the server console
